include/tcframe/spec/variable/WhitespaceManipulator.hpp

```diff
@@ -46,6 +46,12 @@ class WhitespaceManipulator {
         }
     }
 
+    static void ensureEof(istream* in, const string& context) {
```

Comment:
I used this overload instead if there is no last variable before EOF. (for example, when the expected OutputFormat() is just empty but there is an output produced by the solution, it just says "Expected <EOF>" instead of "Expected <EOF> after ".